cmd/secret/main.go

```diff
@@ -1,14 +1,21 @@
 package main
 
 import (
+	"context"
 	"fmt"
+
+	"github.com/go-itools-internship/go-secret/cmd/secret/cmd"
 )
 
 var (
 	revision = "unknown"
 )
 
 func main() {
-	fmt.Printf("secret %s\n", revision)
 	fmt.Println("Hi from go-secret!")
```

Comment:
I guess this line could be remove since we have the command prepared with details.